<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Me</class>
 <widget class="QMainWindow" name="Me">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(190, 242, 255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>311</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_Name">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Username">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Email">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Address">
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Phone">
       <property name="text">
        <string>Phone</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Creer">
       <property name="text">
        <string>Career</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_EditProfile">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(20,176,191);</string>
       </property>
       <property name="text">
        <string>Edit profile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_AreYouCompany">
       <property name="text">
        <string>Are you a company?</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_Picture">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="text">
        <string>Your prifile picture</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(20,176,191);</string>
       </property>
       <property name="text">
        <string>‚úç</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
